<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PTUSplitter</class>
 <widget class="QWidget" name="PTUSplitter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PTU-SPlitter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Progress</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Output folder</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="suffix">
        <string> k</string>
       </property>
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>300</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Split</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Photons per file</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="5">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>File</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QToolButton" name="toolButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="placeholderText">
      <string>Output folder name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit">
     <property name="placeholderText">
      <string>Drop file here</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="1" column="5">
    <widget class="QToolButton" name="toolButton_2">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
